marklogic-data-hub-central/ui/src/components/steps/steps.tsx

```diff
@@ -218,63 +215,60 @@ const Steps: React.FC<Props> = (props) => {
   };
 
   return <Modal
```

Comment:
The header here looks lighter than before. Can we try bolding it? ("New Loading Step")
PR: 
<img width="1213" alt="Screen Shot 2021-11-04 at 2 40 38 PM" src="https://user-images.githubusercontent.com/18374593/140424109-07363812-7bf3-4e3c-bd1c-18119d9ae159.png">

Nightly:
<img width="1034" alt="Screen Shot 2021-11-04 at 2 40 29 PM" src="https://user-images.githubusercontent.com/18374593/140424173-580b6fa2-b3ef-4259-b04f-d9b949c0a357.png">

Reply:
This was standardized the same font that create flow dialog and was checked with UI/UX (Julia)